pypeclub/OpenPype
# AfterEffects: Allow configuration of default variant via Settings

## Brief description
Variant name for subset name creation was hardcoded. Now it is possible to configure it via Settings.

## Description
Implemented as a part of move to New Publisher.
Multiple variants possible.

## Additional info
Depends on https://github.com/pypeclub/OpenPype/pull/2838. Should be tested by merging to #2838 as that branch might contain new updates/fixes for general behavior in NP.

## Testing notes:
1. Configure required value in `Settings:project_settings/aftereffects/create/RenderCreator`
2. Open render creator in NP, check value in the middle bottom (just above gray colored final subset name)
